packages/testcontainers/src/container-runtime/clients/container/docker-container-client.ts

```diff
@@ -264,6 +264,17 @@ export class DockerContainerClient implements ContainerClient {
     }
   }
 
+  async commit(container: Container, opts?: CommitOptions): Promise<void> {
+    try {
+      log.debug(`Committing container...`, { containerId: container.id });
+      await container.commit(opts);
```

Comment:
We should provide labels as we do here, perhaps let the user specify whether to `deleteOnExit`, so that the image is cleaned up by the resource reaper when the tests complete: https://github.com/testcontainers/testcontainers-node/blob/5817d695fe370168ec25a52f9ee1b59b1a3775e1/packages/testcontainers/src/generic-container/generic-container-builder.ts#L66-L82

Reply:
thank you for calling this out!  In order to get this to work, I had to also expose the `changes` option because by default it was actually saving the label with the sessionId to the committed image.  Now, we'll apply the change command to unset the label if `deleteOnExit` is true (added a test case for this as well).  Lmk what you think / if you have concerns with the approach.